### DISCUSSION TITLE
Do not share users with the LTSP server and use home directory from the image

### DISCUSSION THREAD

**Discussion by r33int** [ltsp/ltsp#669] [Q&A]
Hi,

I am trying to set up an LTSP terminal server, using VM images. I got everything up and running, but the default behavior, where the users and home folder are shared with the server does not fit my needs, and I couldn't find anything online about it.

I want to make it so that the user(s) created while creating the VM image are kept as is, and that the home directory is also part of the image: I want to be able to customize user settings directly on the image, and that any change made by the user is forgotten after shutting down the client, and that the client is not able to write any changes to the server.

I wanted to know if there was any way to achieve the following.

Thanks ahead of time!

**Comment by alkisg** [comment]
Sure, just read the [OMIT_IMAGE_EXCLUDES](https://ltsp.org/man/ltsp.conf/#OMIT_IMAGE_EXCLUDES) parameter to include the /home from the image, and the [PWMERGE*](https://ltsp.org/man/ltsp.conf/#PRE_APPLET_x) parameters in order to keep the user accounts from the image.

**Reply by r33int** [reply]
Hi, thank for for your reply. I tried to add
`OMIT_IMAGE_EXCLUDES="home/*"` to LTSP.conf. I have noticed that with 2 users created, the home directory of the user not logged in is still is the one from ths image, but whenever I log in at that second user, its home directory still gets mounted from SSHFS on the server. I am also confused about the usage of the PWMERGE parameter. Did I miss something here? Thanks

**Comment by alkisg** [comment]
You need to properly use PWMERGE to tell LTSP that these users should **not** use pamltsp (i.e. to not use SSH for authentication and SSHFS for /home/username).
Essentially you want to set the PWMERGE variables to "everything" so that all user accounts are kept both on image creation and when the LTSP clients boots, i.e. they should be placed under `[common]` in ltsp.conf, not under `[clients]`.

**Comment by r33int** [comment]
I tried setting the PWMERGE variables for the user I want to use locally:

```
[common]
PWMERGE_DUR="user"
PWMERGE_DGR="user"
PWMERGE_SUR="user"
PWMERGE_SGR="user"
```

I noticed that the image is no longer getting populated with other users from the server. However the "user" user cannot login with the password set in the image when it does not exist on the LTSP server, and its home directory still gets mounted from sshfs.
I am really sorry if I am not understanding something, and I am very grateful for your help so far.

**Comment by alkisg** [comment]
@r33int, you may come to the [chat room](http://ltsp.org/advanced/chat-room/) for live support on this if you want, as it'll be faster than trying to reproduce your setup locally.
